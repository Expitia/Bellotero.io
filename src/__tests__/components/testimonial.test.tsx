import { shallow } from 'enzyme'
import React from 'react'
import Testimonial from '../../components/testimonial'

describe('Testimonial', () => {
  it('should render Testimonial component', () => {
    const component = shallow(
      <Testimonial
        title="Titulo de prueba"
        reviews={[
          {
            name: 'Juan Prueba',
            position: 'Probador profesional',
            comment: 'Fue una prueba muy placentera',
          },
        ]}
      />,
    )

    expect(component).toMatchSnapshot()
  })
})
