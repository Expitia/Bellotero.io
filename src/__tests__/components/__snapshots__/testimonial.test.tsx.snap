// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonial should render Testimonial component 1`] = `
<div
  className="testimonial container d-flex flex-column"
>
  <div
    className="bellotero-title mb-5"
  >
    Titulo de prueba
  </div>
</div>
`;
